src/tests/multiDomainFirstParties.js

```diff
@@ -0,0 +1,14 @@
+(function(){
```

Comment:
We actually do already have tests for MDFP, or at least a test, in [tests/utils.js](https://github.com/EFForg/privacybadger/blob/d217a28ee3188d51797f909763ae7d108a1ee63c/src/tests/tests/utils.js#L95-L108).